src/ilcdlib/epd/reader.py

```diff
@@ -401,6 +402,26 @@ def get_openepd_compliance(self, lang: LangDef, base_url: str | None = None) ->
             )
         return result
 
+    def _get_all_verifiers(self, lang: LangDef) -> list[ValidationDto]:
+        """Return list of all verifiers with type."""
+        reader = self.get_validation_reader()
+        if reader is None:
+            return []
+        return reader.get_validations(lang)
+
+    def get_verifiers(self, lang: LangDef) -> list[ValidationDto]:
```

Comment:
`_get_all_verifiers` is not needed any more. Move logic from `_get_all_verifiers` to this method